pynucastro/rates/library.py

```diff
@@ -25,6 +27,58 @@ def list_known_rates():
                 print(f"                                 : {r}")
 
 
+def _rate_name_to_nuc(name):
+
+    # first try to interpret name as A(x,y)B
+    rate_str = re.compile(r"([A-Za-z0-9]+)\(([A-Za-z0-9]*),([A-Za-z0-9]*)\)([A-Za-z0-9]+)",
+                          re.IGNORECASE)
+    nucs = rate_str.search(name)
+    try:
+        _r = [nucs.group(1), nucs.group(2)]
+        _p = [nucs.group(3), nucs.group(4)]
+    except AttributeError:
+        return None
+
+    # now try to make nuclei objects.
+    reactants = []
+    for nuc in _r:
+        if nuc == "":
+            continue
+        try:
+            n = Nucleus(nuc)
```

Comment:
We get an Assertion Error here if we try to use underscores in the shortname, like in`reaclib_library.get_rate_by_name("f20(_,e)ne20")`.

Reply:
ah good catch.  I'll fix that in a minute

Reply:
okay, these all work now:
```
r = rl.get_rate_by_name("f20(_,e)ne20")
r = rl.get_rate_by_name("f20(,e)ne20")
r = rl.get_rate_by_name("f20(,)ne20")
```